yarn.lock

```diff
@@ -5,13 +5,12 @@ __metadata:
   version: 5
   cacheKey: 8
 
-"@ampproject/remapping@npm:^2.0.0":
-  version: 2.0.2
-  resolution: "@ampproject/remapping@npm:2.0.2"
+"@ampproject/remapping@npm:^2.0.0, @ampproject/remapping@npm:^2.1.0":
```

Comment:
I don't get yarn, but we need v2.1.0 to ensure the new `ctx` param is passed to the remapping loader. Will just the change in the `package.json` be enough?

Reply:
Yeah it's enough